config/crd/bases/mongodb.com_mongodbsearch.yaml

```diff
@@ -117,6 +115,12 @@ spec:
                             the Pod where this field is used. It makes that resource available
                             inside a container.
                           type: string
+                        request:
+                          description: |-
+                            Request is the name chosen for a request in the referenced claim.
+                            If empty, everything from the claim is made available, otherwise
+                            only the result of this request.
+                          type: string
```

Comment:
Even though the `request` field was already part of `ResourceRequirements` type in the earlier version of `k8s.io/api` it looks like we didn't generate the manifest lately. Because I have not made any changes to our types and `make manifests` resulted into this change.